envs/conda_build_config.yaml

```diff
@@ -112,6 +118,8 @@ keras_preprocessing:
   - 1.1.*
 leveldb:
   - 1.20.*
+libdate:
```

Comment:
Maybe set this one to just 3.*
It looks like this is getting included into c++20, so it's likely there will only be 3.0.x updates at most with libdate, but in case there is a 3.1 let's allow that.

Reply:
I kept it 3.0 because our recipe for libdate builds 3.0.1. In case there is a 3.1 version, we can update this too when we update the recipe for it. I'm okay with changing it to 3.*.